cmd/root.go

```diff
@@ -62,3 +72,35 @@ func validateKey(key string) error {
 	}
 	return nil
 }
+
+func getSecretStore() store.Store {
+	backend, ok := os.LookupEnv(BackendEnvVar)
+	if !ok {
+		backend = SSMBackend
+	}
+
+	backend = strings.ToUpper(backend)
+	if !stringInSlice(backend, Backends) {
+		// TODO: warn user
+		backend = SSMBackend
+	}
+
+	switch backend {
+	case SSMBackend:
+		return store.NewSSMStore(numRetries)
+	case S3Backend:
+		return store.NewS3Store(numRetries)
+	}
+
+	// This line is unreachable, but necessary to satisfy the compiler
+	panic("unreachable")
```

Comment:
I think this can be simplified? Eg
```
    var store Store
    switch backendEnv {
        case "s3":
            store = S3Store{}
        case "ssm":
            fallthrough
        default:
            store = SSMStore{}
    }
    return store
```

Reply:
Good call, I like that

Reply:
ff5fb8f